ivy/src/main/scala/sbt/ConvertResolver.scala

```diff
@@ -94,37 +96,34 @@ private[sbt] object ConvertResolver {
     }
   }
 
-  private[sbt] val USE_AETHER_PROPERTY = "sbt.use.aether"
-  private def isUseAetherForResolution(settings: IvySettings): Boolean =
-    settings.getVariable(USE_AETHER_PROPERTY) == "true"
+  /** Converts the given sbt resolver into an Ivy resolver. */
+  @deprecated("0.13.8", "Use the variant with updateOptions")
+  def apply(r: Resolver, settings: IvySettings, log: Logger): DependencyResolver =
+    apply(r, settings, UpdateOptions(), log)
+
+  /** Converts the given sbt resolver into an Ivy resolver. */
+  def apply(r: Resolver, settings: IvySettings, updateOptions: UpdateOptions, log: Logger): DependencyResolver =
+    (updateOptions.resolverConverter orElse defaultConvert)((r, settings, log))
 
-  /** Converts the given sbt resolver into an Ivy resolver..*/
-  def apply(r: Resolver, settings: IvySettings, log: Logger) =
-    {
+  /** The default implementation of converter. */
+  lazy val defaultConvert: ResolverConverter = {
+    case (r, settings, log) =>
       r match {
         case repo: MavenRepository =>
           {
-            if (isUseAetherForResolution(settings)) {
-              repo match {
-                case cache: MavenCache => new org.apache.ivy.plugins.resolver.MavenCacheRepositoryResolver(cache, settings)
-                case _                 => new org.apache.ivy.plugins.resolver.MavenRemoteRepositoryResolver(repo, settings)
+            val pattern = Collections.singletonList(Resolver.resolvePattern(repo.root, Resolver.mavenStyleBasePattern))
+            final class PluginCapableResolver extends IBiblioResolver with ChecksumFriendlyURLResolver with DescriptorRequired {
```

Comment:
This needs the CustomMavenRepo tag or MakePom will start injected maven central into pom.xml files silently.